### DISCUSSION TITLE
1.1.4 m1 맥 OnRenderImage 사용 오류

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#387] [Issue Report (Legacy)]
OnRenderImage(source: RenderTexture, destination: RenderTexture)

인탤맥의 경우 정상작동하지만
M1맥의 경우 택스처 포맷 7번 사용시 개발환경에서 오류
2번 포맷도 개발환경에서 정상출력 안됨

7번은 모바일에서 정상출력

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 지금도 같은 오류가 발생하나요? 상세하게 리포트 해주시면 도움을 드릴 수 있을것 같습니다.
